docs/book/getting-started/zenml-pro/README.md

```diff
@@ -39,62 +39,16 @@ To try ZenML Pro or to learn more [book a call](https://www.zenml.io/book-your-d
 | **Pipeline Execution** | Run pipelines via SDK/CLI | Run pipelines from the dashboard, manage schedules via UI, [triggers](https://docs.zenml.io/concepts/triggers) |
```

Comment:
in the line above it points to MCP and ACP docs that should be different. rather point to https://docs.zenml.io/concepts/artifacts and https://docs.zenml.io/concepts/models